src/test/fileUtils.spec.ts

```diff
@@ -156,19 +155,31 @@ describe("fileUtils", () => {
       }`
       const result = parseSpecification(specificationContent)
       const expected: Specification = {
-        "name": "tool",
-        "version": "10",
-        "patterns": [
+        name: "tool",
+        version: "10",
+        patterns: [
           {
-            "patternId": "a-patternId",
-            "level": "Warning",
-            "category": "CodeStyle",
-            "parameters": [],
-            "enabled": false
+            patternId: "a-patternId",
+            level: "Warning",
+            category: "CodeStyle",
+            parameters: [],
+            enabled: false
           }
         ]
       }
       deepStrictEqual(result, expected)
     })
   })
+  it("should parse a codacyrc file with options", () => {
```

Comment:
To be coherent with the remaining tests, `options` should be added to the first test and here it should test without `options` set.